.github/workflows/test-runner-nightly-perf.yml

```diff
@@ -0,0 +1,54 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Sample Nightly Perf Tests
+
+on:
+  workflow_dispatch: {}
+  schedule:
+    - cron: '0 14 * * *' # run at 2 PM UTC, 9 AM EST.
+
+jobs:
+  test-runner-integration:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up AdoptOpenJDK 11
+        uses: joschi/setup-jdk@v2
+        with:
+          java-version: 11
+      - name: Get Vault token
+        id: vault-token-step
+        env:
+          VAULT_ADDR: https://clotho.broadinstitute.org:8200
+        run: |
+          VAULT_TOKEN=$(docker run --rm --cap-add IPC_LOCK \
```

Comment:
Can you pull this docker business into its own script or action? It looks like we're dropping two abstraction levels from the surrounding config. I understand if GH actions doesn't have great support here.

Reply:
Right now, this is what everybody's else is doing. I'll create a separate ticket for future discussions across teams.